src/ArborX_LinearBVH.hpp

```diff
@@ -185,7 +185,7 @@ BasicBoundingVolumeHierarchy<MemorySpace, BoundingVolume, Enable>::
   // map primitives from multidimensional domain to one-dimensional interval
   using LinearOrderingValueType = Kokkos::detected_t<
       Details::SpaceFillingCurveProjectionArchetypeExpression,
-      SpaceFillingCurve, Point>;
+      SpaceFillingCurve, Box, Point>;
```

Comment:
Can't you deduce `Box` why would we need to specify it?

Reply:
There's nothing to deduce it from, both it and point are argument to `operator()`.

Reply:
If you get a `Point` you could assume `Box`, with`Exp::Point<DIM>` you could chose `Exp::Box<DIM>`

Reply:
You can't. A user may run access traits with old Point or Box, but we will always use hyper-Box for the scene bounding box.

Reply:
Had in-person discussion. Decided to just check one type of geometries.